ContainerLogWatchTest fails sometimes

### Describe the bug
ContainerLogWatchTest fail sometimes

### Che version
<!-- (if workspace is running, version can be obtained with help/about menu) -->
 - [X] latest
 - [ ] nightly
 - [ ] other: please specify

### Steps to reproduce

unknown

### Expected behavior
no error

### Eclipse Che Logs

```
2020-05-20T07:02:12.7766987Z Call: INSERT INTO che_k8s_runtime (workspace_id, env_name, namespace, owner_id, status) VALUES (?, ?, ?, ?, ?)
2020-05-20T07:02:12.7767310Z 	bind => [5 parameters bound]
2020-05-20T07:02:12.7767942Z Query: InsertObjectQuery(KubernetesRuntimeState{workspaceId='wsId1wnposkg', envName='defEnv', ownerId='idf35upfqe', namespace='namespace2gjpp', status=RUNNING})
2020-05-20T07:02:12.8895153Z [EL Info]: connection: 2020-05-20 07:02:12.888--ServerSession(2101313099)--/file:/home/runner/work/che/che/infrastructures/kubernetes/target/test-classes/_test logout successful
2020-05-20T07:02:13.4516806Z [ERROR] Tests run: 708, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 14.589 s <<< FAILURE! - in TestSuite
2020-05-20T07:02:13.4528592Z [ERROR] testCloseOfOutputStream(org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log.ContainerLogWatchTest)  Time elapsed: 0.011 s  <<< FAILURE!
2020-05-20T07:02:13.4533245Z org.mockito.exceptions.verification.WantedButNotInvoked: 
2020-05-20T07:02:13.4534381Z 
2020-05-20T07:02:13.4537659Z Wanted but not invoked:
2020-05-20T07:02:13.4541675Z eventsPublisher.sendWatchLogStoppedEvent(
2020-05-20T07:02:13.4544273Z     <any java.lang.String>
2020-05-20T07:02:13.4545525Z );
2020-05-20T07:02:13.4555986Z -> at org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log.ContainerLogWatchTest.testCloseOfOutputStream(ContainerLogWatchTest.java:202)
2020-05-20T07:02:13.4557415Z 
2020-05-20T07:02:13.4562208Z However, there was exactly 1 interaction with this mock:
2020-05-20T07:02:13.4565634Z eventsPublisher.sendWatchLogStartedEvent(
2020-05-20T07:02:13.4569035Z     "namespace123:pod123:containre123"
2020-05-20T07:02:13.4570253Z );
2020-05-20T07:02:13.4581413Z -> at org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log.ContainerLogWatch.run(ContainerLogWatch.java:101)
2020-05-20T07:02:13.4582702Z 
2020-05-20T07:02:13.4583819Z 
2020-05-20T07:02:13.4595048Z 	at org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log.ContainerLogWatchTest.testCloseOfOutputStream(ContainerLogWatchTest.java:202)
2020-05-20T07:02:13.4596429Z 
2020-05-20T07:02:13.4606753Z [ERROR] testLimitInputStreamBytes(org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log.ContainerLogWatchTest)  Time elapsed: 0.002 s  <<< FAILURE!
2020-05-20T07:02:13.4611441Z org.mockito.exceptions.verification.TooManyActualInvocations: 
2020-05-20T07:02:13.4612535Z 
2020-05-20T07:02:13.4615967Z eventsPublisher.sendWatchLogStoppedEvent(
2020-05-20T07:02:13.4618578Z     <any java.lang.String>
2020-05-20T07:02:13.4619841Z );
2020-05-20T07:02:13.4622003Z Wanted 1 time:
2020-05-20T07:02:13.4633840Z -> at org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log.ContainerLogWatchTest.testLimitInputStreamBytes(ContainerLogWatchTest.java:122)
2020-05-20T07:02:13.4636099Z But was 2 times:
2020-05-20T07:02:13.4644817Z -> at org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log.ContainerLogWatch.run(ContainerLogWatch.java:138)
2020-05-20T07:02:13.4652952Z -> at org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log.ContainerLogWatch.run(ContainerLogWatch.java:138)
2020-05-20T07:02:13.4654295Z 
2020-05-20T07:02:13.4655394Z 
2020-05-20T07:02:13.4664810Z 	at org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log.ContainerLogWatchTest.testLimitInputStreamBytes(ContainerLogWatchTest.java:122)
2020-05-20T07:02:13.4665951Z 
2020-05-20T07:02:13.8054788Z [INFO] 
2020-05-20T07:02:13.8055002Z [INFO] Results:
2020-05-20T07:02:13.8055305Z [INFO] 
2020-05-20T07:02:13.8065391Z [ERROR] Failures: 
2020-05-20T07:02:13.8065772Z [ERROR]   ContainerLogWatchTest.testCloseOfOutputStream:202 
2020-05-20T07:02:13.8065994Z Wanted but not invoked:
2020-05-20T07:02:13.8066207Z eventsPublisher.sendWatchLogStoppedEvent(
2020-05-20T07:02:13.8066463Z     <any java.lang.String>
2020-05-20T07:02:13.8067199Z );
2020-05-20T07:02:13.8068093Z -> at org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log.ContainerLogWatchTest.testCloseOfOutputStream(ContainerLogWatchTest.java:202)
2020-05-20T07:02:13.8068335Z 
2020-05-20T07:02:13.8068560Z However, there was exactly 1 interaction with this mock:
2020-05-20T07:02:13.8068769Z eventsPublisher.sendWatchLogStartedEvent(
2020-05-20T07:02:13.8068981Z     "namespace123:pod123:containre123"
2020-05-20T07:02:13.8069192Z );
2020-05-20T07:02:13.8069752Z -> at org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log.ContainerLogWatch.run(ContainerLogWatch.java:101)
2020-05-20T07:02:13.8069960Z 
2020-05-20T07:02:13.8070112Z 
2020-05-20T07:02:13.8070351Z [ERROR]   ContainerLogWatchTest.testLimitInputStreamBytes:122 
2020-05-20T07:02:13.8070588Z eventsPublisher.sendWatchLogStoppedEvent(
2020-05-20T07:02:13.8070777Z     <any java.lang.String>
2020-05-20T07:02:13.8070977Z );
2020-05-20T07:02:13.8071156Z Wanted 1 time:
2020-05-20T07:02:13.8071903Z -> at org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log.ContainerLogWatchTest.testLimitInputStreamBytes(ContainerLogWatchTest.java:122)
2020-05-20T07:02:13.8072364Z But was 2 times:
2020-05-20T07:02:13.8073461Z -> at org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log.ContainerLogWatch.run(ContainerLogWatch.java:138)
2020-05-20T07:02:13.8074377Z -> at org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log.ContainerLogWatch.run(ContainerLogWatch.java:138)
2020-05-20T07:02:13.8074591Z 
2020-05-20T07:02:13.8074777Z 
```

--- a/infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/log/ContainerLogWatchTest.java
+++ b/infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/log/ContainerLogWatchTest.java
@@ -199,7 +199,7 @@ public class ContainerLogWatchTest {
 
     // verify events were properly fired
     verify(eventsPublisher, times(1)).sendWatchLogStartedEvent(any(String.class));
-    verify(eventsPublisher, times(1)).sendWatchLogStoppedEvent(any(String.class));
+    verify(eventsPublisher, timeout(1000).times(1)).sendWatchLogStoppedEvent(any(String.class));
   }
 
   @Test

